### ISSUE TITLE
Error on startup - Linux - libXrandr.so.2

### ISSUE THREAD

**Issue by KeitIG** [atom/atom#4865]
I got an error after updating atom today:

/opt/atom/atom: error while loading shared libraries: libXrandr.so.2: cannot open shared object file: No such file or directory

Atom does not launch anymore

**Comment by jeekajoo** [comment]
since release 0.166, I'm getting this:

```javascript
/opt/atom $ ./atom                                                
zsh: no such file or directory: ./atom
```

**Comment by KeitIG** [comment]
Well, I think it was my bad, I had a conflict with webupd8 atom repository, which seems to share a corrupted atom installation.

@jeekajoo what about `atom` instead of `./atom` ?

**Comment by jeekajoo** [comment]
same with just `atom`
```javascript
nohup: failed to run command ‘/opt/atom/atom’: No such file or directory
```
I'm also using last atom version from webupd8
but yep, I guess it's corrupted.

**Comment by KeitIG** [comment]
What I did: 

I removed the webupd8 repo, then remove atom, and reinstall atom with the .deb found on Github

**Comment by jeekajoo** [comment]
fixed in last release (0.166.0-3~webupd8~0)
sorry for the noise guys
